man/systemd.exec.xml

```diff
@@ -74,6 +74,11 @@
     execution specific configuration options are configured in the
     [Service], [Socket], [Mount], or [Swap] sections, depending on the
     unit type.</para>
+
+    <para>In addition, options which control resources through cgroups
```

Comment:
i am pretty sure we should avoid using the term "cgroups" like this. In prose we should spell this out as "Control Groups" or so.